src/components/Dashboard/FilingHistoryList/bodies/FutureEffectivePending.vue

```diff
@@ -64,7 +64,7 @@ export default class FutureEffectivePending extends Vue {
     if (EnumUtilities.isTypeContinuationIn(this.filing)) {
       return {
         subtitle: 'Incorporation Pending',
-        filingLabel: 'continuation in'
+        filingLabel: 'continuation application'
```

Comment:
What does this look like for a future effective continuation application?

I'm looking at "incorporation" on line 49 and it doesn't say "application". Maybe this should be the same. It depends on how this filing label is used.

Also, please ask Andy about the subtitle on line 66. A continuation in **is** a type of incorporation but Andy may want something less generic here.

Reply:
Currently I can't access the filing history page (in dev), is there an issue?:
![image](https://github.com/bcgov/business-filings-ui/assets/122495122/cdd8b206-6473-4fe7-a6bd-760b8c4ccbfe)
However, I looked at the screenshots I previously saved, the subtitle and filing label are not set properly:
![image](https://github.com/bcgov/business-filings-ui/assets/122495122/821ce4ee-26ac-4411-a36f-111158ae1e6e)
They all go to the last default value on line 79:
![image](https://github.com/bcgov/business-filings-ui/assets/122495122/73b0ef54-98eb-483d-a6b6-5fb06bd79463)

Reply:
Oh, you found a different bug :)

Can you fix it and then we'll see if the title and label make sense?

Reply:
Checked the default values are not being used